src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -252,9 +245,6 @@ public virtual void CopyTo(Stream destination, int bufferSize)
         public virtual void Close()
         {
             /* These are correct, but we'd have to fix PipeStream & NetworkStream very carefully.
-            Contract.Ensures(CanRead == false);
-            Contract.Ensures(CanWrite == false);
-            Contract.Ensures(CanSeek == false);
             */
```

Comment:
Shouldn't the comment be deleted, too?

Reply:
Thanks, fixed. I reviewed `git diff | grep "^-"` etc so I missed the context.